sdk/storage/azure-storage-files-shares/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Release History
 
+## 12.0.0-beta.8 (Unreleased)
+
 ## 12.0.0-beta.7 (2021-02-04)
```

Comment:
Where is the release tag for files-shares? https://github.com/Azure/azure-sdk-for-cpp/tags

Reply:
The pipeline that was run failed because it didn't include this change. We need to re-run it, and I am waiting for sync-ing with the storage team to help sort it out.
https://github.com/Azure/azure-sdk-for-cpp/pull/1595

See this:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=717067&view=results